Cargo.toml

```diff
@@ -18,6 +18,10 @@ path = "src/client.rs"
 name = "civkit-sample"
 path = "src/sample.rs"
 
+[[bin]]
+name = "parser"
+path = "src/parser.rs"
```

Comment:
Yes this comment it can be integrated into `civkitd`. Once civkit-cli and civkit-sample becomes more complex, a toml parser can be added too.

Reply:
Removed as it now runs when you run civkitd.